src/libnnstreamer-edge/nnstreamer-edge-aitt.c

```diff
@@ -221,7 +221,7 @@ _nns_edge_invoke_event_cb (nns_edge_handle_s * eh, nns_edge_event_e event,
  */
 static void
 aitt_cb_message_arrived (aitt_msg_h msg_handle, const void *msg,
-    size_t msg_len, void *user_data)
+    uint64_t msg_len, void *user_data)
```

Comment:
Q: does AITT use 64b data type for message size? (IMO size_t > unit32_t)

Reply:
or size_t > unsigned int

Reply:
Thanks. I only modified the data which is related with communication.